.github/workflows/checkMetaMaskLatest.yml

```diff
@@ -0,0 +1,33 @@
+name: Check MetaMask is up to date
+
+on:
+  pull_request:
```

Comment:
Will this be required to pass before merging a PR?
Because I believe this will fail from time to time in our PRs and we won't update it on these pull requests.